language/use-cases/prod-catalog-enrichment/genai_prod_catalog_enrichment.ipynb

```diff
@@ -0,0 +1,2071 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #42.        vertexai.init(project=project, location="us-central1")
</pre><p>Put this init closer to the top of the notebook. </p><p> </p><p>Also make the PROJECT_ID into a constant /parameter hat can be easily set at the beginning.</p><p> </p><p>See the gemini intro notebook for an example.</p><p> </p><p><a href="https://github.com/GoogleCloudPlatform/generative-ai/blob/main/gemini/getting-started/intro_gemini_python.ipynb" target="_blank">https://github.com/GoogleCloudPlatform/generative-ai/blob/main/gemini/getting-started/intro_gemini_python.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>The Project_ID is being set only once during the runner function init and then passed into the lower down functions. You are right we could remove the project id as parameters from the lower down functions and have it as global, but thinking w.r.t a code repo perspective and abstraction . I would be tempted to keep it the way it is ATM. This to keep clear handshakes between different functions and PROJECT ID will still just need to be declared once as you suggested earlier.</p>